scripts/bintray.sh

```diff
@@ -3,8 +3,21 @@ set -e
 
 OUTPUT=${1:-out}
 
+if curl -s -I "https://api.bintray.com/repos/$BINTRAY_PROJECT" | grep -q 404; then
+    echo "package repository not found, creating repository"
+    curl -u$BINTRAY_USER:$BINTRAY_API_KEY -X POST -I "https://api.bintray.com/repos/$BINTRAY_PROJECT"
+fi
+
 for PKG in $(ls $OUTPUT); do
+    if curl -s -I "https://api.bintray.com/packages/$BINTRAY_PROJECT/$PKG" | grep -q 404; then
+        echo "package $PKG not found, creating package "
+        PKG_NAME=$(echo $PKG | sed -E "s|^(([a-zA-Z\-]+)-([0-9\.\-]+)\..*)$|\2|")
+        PKG_METADATA='{"name": "$PKG_NAME", "licenses": ["Apache-2.0"], "vcs_url": "https://github.com/mantl-pkg/stub-url"}'
```

Comment:
I'd go for the root of the repo. We dont have a `repo <-> pkg-name` mapping, say for `mesos-master` and `mesos-slave` there are no git repositories.
Now which 'root' to choose is the question: `https://github.com/mantl-pkg/` or `https://github.com/asteris-llc/mantl-packaging` ?